src/modules/mesh/AnimationModule.js

```diff
@@ -0,0 +1,31 @@
+import {
+  AnimationMixer,
+  AnimationClip,
+  Clock
+} from 'three';
+
+export class AnimationModule {
+  constructor(params = {}) {
+    this.params = Object.assign(params);
+    this.clock = new Clock();
+  }
+
+  play(clipName) {
+    const clip = AnimationClip.findByName(this.clips, clipName);
+    const action = this.mixer.clipAction(clip);
+    action.play();
+  }
+
+  update() {
+    if (this.mixer) this.mixer.update(this.clock.getDelta());
+  }
+
```

Comment:
Let's add: 

```js
manager(manager) {
  manager.define('animation');
}
```

That way it's API can be used by other modules with `manager.use('animation')  <- returns module scope`